<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>RangeWidget</class>
 <widget class="QWidget" name="RangeWidget">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>512</width>
    <height>272</height>
   </rect>
  </property>
  <property name="sizePolicy">
   <sizepolicy hsizetype="Expanding" vsizetype="Expanding">
    <horstretch>0</horstretch>
    <verstretch>0</verstretch>
   </sizepolicy>
  </property>
  <property name="minimumSize">
   <size>
    <width>0</width>
    <height>0</height>
   </size>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout">
   <property name="leftMargin">
    <number>0</number>
   </property>
   <property name="topMargin">
    <number>0</number>
   </property>
   <property name="rightMargin">
    <number>0</number>
   </property>
   <property name="bottomMargin">
    <number>0</number>
   </property>
   <item>
    <layout class="QHBoxLayout" name="horizontalLayout">
     <item>
      <widget class="QLabel" name="label">
       <property name="text">
        <string>Data range</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QDoubleSpinBox" name="minDataRange">
       <property name="enabled">
        <bool>false</bool>
       </property>
       <property name="decimals">
        <number>4</number>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QDoubleSpinBox" name="maxDataRange">
       <property name="enabled">
        <bool>false</bool>
       </property>
       <property name="decimals">
        <number>4</number>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QPushButton" name="fitToDataButton">
       <property name="enabled">
        <bool>false</bool>
       </property>
       <property name="text">
        <string>Fit to data</string>
       </property>
      </widget>
     </item>
    </layout>
   </item>
   <item>
    <widget class="QQuickWidget" name="background">
     <property name="resizeMode">
      <enum>QQuickWidget::SizeRootObjectToView</enum>
     </property>
     <property name="source">
      <url>
       <string>qrc:/qml/RangeWidget.qml</string>
      </url>
     </property>
    </widget>
   </item>
  </layout>
 </widget>
 <customwidgets>
  <customwidget>
   <class>QQuickWidget</class>
   <extends>QWidget</extends>
   <header>QtQuickWidgets/QQuickWidget</header>
  </customwidget>
 </customwidgets>
 <resources/>
 <connections/>
</ui>
